apps/client/src/components/Visual.js

```diff
@@ -1,9 +1,12 @@
-import React from "react";
-import { Figure, Img, FigCaption } from "./VisualStyles";
+import React, { useState } from "react";
+import { StyledCard, Img, FigCaption } from "./VisualStyles";
 
-export default ({ title, ...rest }) => (
-  <Figure>
-    <Img {...rest} />
-    {title && <FigCaption>{title}</FigCaption>}
-  </Figure>
-);
+export default ({ title, ...rest }) => {
+  const [loading, setLoading] = useState(true);
+  return (
+    <StyledCard isLoading={loading}>
```

Comment:
I understand you choose `Card` because it has a `isLoading` prop, but I think `Card`  is not suitable in this place because a Card is not an image container. Can you think of another solution?

Reply:
Update: you can use `perceivedLoading` from `asc-ui` to style the loading

Reply:
Nicee